docs/design.md

```diff
@@ -1,25 +1,31 @@
 # OpenTelemetry Collector Architecture
 
-This document describes the architecture design and implementation of
+This document describes the architecture design and implementation of the
 OpenTelemetry Collector.
 
 ## Summary
 
-OpenTelemetry Collector is an executable that allows to receive telemetry data, optionally transform it and send the data further.
+OpenTelemetry Collector is an executable that can receive telemetry data, optionally process it, and export it further.
 
-The Collector supports several popular open-source protocols for telemetry data receiving and sending as well as offering a pluggable architecture for adding more protocols.
+The Collector supports several popular open-source protocols for receiving and sending telemetry data as well as offering a pluggable architecture for adding more protocols.
 
-Data receiving, transformation and sending is done using Pipelines. The Collector can be configured to have one or more Pipelines. Each Pipeline includes a set of Receivers that receive the data, a series of optional Processors that get the data from receivers and transform it and a set of Exporters which get the data from the Processors and send it further outside the Collector. The same receiver can feed data to multiple Pipelines and multiple pipelines can feed data into the same Exporter.
+Data receiving, processing, and exporting is done using [Pipelines](#pipelines). The Collector can be configured to have one or more pipelines. Each pipeline includes:
+
+- a set of [Receivers](#receivers) that receive the data
+- a series of optional [Processors](#processors) that get the data from receivers and process it
+- a set of [Exporters](#exporters) which get the data from processors and send it further outside the Collector.
+
+The same receiver can feed data to multiple pipelines and multiple pipelines can feed data into the same Exporter.
```

Comment:
```suggestion
The same receiver can be included in multiple pipelines and multiple pipelines can include the same Exporter.
```

This read as if receivers and exporters were separate from pipelines, which I don't think is the intent.